lxd/instance/drivers/driver_common.go

```diff
@@ -589,6 +589,45 @@ func (d *common) restartCommon(inst instance.Instance, timeout time.Duration) er
 	return nil
 }
 
+// rebuildCommon handles the common part of instance rebuilds.
+func (d *common) rebuildCommon(inst instance.Instance, fingerprint string, op *operations.Operation) error {
```

Comment:
Could you rename `fingerprint` to `imgFingerprint` so its clear what it is a fingerprint of please?